packages/reporter/src/formatters/sarif-formatter.ts

```diff
@@ -204,6 +201,20 @@ function buildArtifact(file: ProcessedFile): Artifact {
     },
   };
 }
+function kindConverter(category: string): Result.kind {
```

Comment:
changed

Reply:
This will not work you need to enumerate the cases:


<img width="439" alt="Screen Shot 2022-12-15 at 10 55 37 PM" src="https://user-images.githubusercontent.com/183799/208018437-ae5d6009-2d6e-4e21-8856-02c0a5cb8de9.png">

```suggestion
function kindConverter(category: string): Result.kind {
  switch (category) {
    case 'Warning':
    case 'Suggestion':
    case 'Message':
      return 'review';
    default:
      return 'fail';
  }
}
```

There is also a spelling error in there `Messge` -> `Message`.